.github/workflows/pr-checks.yaml

```diff
@@ -74,15 +60,15 @@ jobs:
 
       - name: Build
         id: gradle-build
-        run: ${GRADLE_EXEC} build
+        run: ${GRADLE_EXEC} assemble --scan
```

Comment:
`build` did everything already and the two other steps below were obsolete.

The split-up in three steps is what we also do in other repos:

- assemple (everything compiles)
- qualityCheck (all checks: spotless, dependency scopes)
- check (run tests)